recipes/diligent-core/all/conanfile.py

```diff
@@ -184,8 +184,13 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Common", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Primitives", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "interface"))
+        self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "Android", "interface"))
+        self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "Apple", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "Basic", "interface"))
+        self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "Emscripten", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "Linux", "interface"))
+        self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "UWP", "interface"))
+        self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "Win32", "interface"))
```

Comment:
Do not close discussions so quickly please (https://github.com/conan-io/conan-center-index/pull/10780#discussion_r872308956).
AFAIK, non-existing includedirs can break downstream project relying on `CMakeDeps` generator.

Reply:
you may keep them alive by creating an empty file

Reply:
I have closed it as @madebr has approved a MR, meaning he agrees with what I have written.